games/g.yaml

```diff
@@ -249,6 +249,23 @@
   updated: 2015-04-06
   url: https://github.com/Lyndir/Gorillas
 
+- name: Gorillas-rs
+  type: clone
+  originals:
+    - Scorched Earth
```

Comment:
the original should be gorilla.bas
https://en.wikipedia.org/wiki/Gorillas_(video_game)

Reply:
Changed! Sorry, I don't know the original game. The gameplay was similar to Scorched Earth and that was my first thought.